### PR TITLE
Add the tests for rhel 7, 8 and suse 15.3

### PR DESCRIPTION
**Pull request by sophia-guo**
related with #410 

Signed-off-by: Sophia Guo <sophia.gwf@gmail.com>

### CODE REVIEW

**Review by karianna — APPROVED** [review]
LGTM

File: linuxNew/jdk/redhat/src/packageTest/java/packaging/RedHatFlavoursWithYum.java [hunk 1]
```diff
@@ -45,6 +45,7 @@
 		return Stream.of(
 			Arguments.of("amazonlinux", "2"),
 			Arguments.of("centos", "7"),
+			Arguments.of("rhel", "7"),
```
Comment (gdams) [inline linuxNew/jdk/redhat/src/packageTest/java/packaging/RedHatFlavoursWithYum.java:48]:
```suggestion
			Arguments.of("registry.access.redhat.com/ubi7/ubi", "latest"),
```

**Comment by gdams** [comment]
@sophia-guo rhel needs to be pulled from the Red Hat container registry (see https://developers.redhat.com/blog/2020/03/24/red-hat-universal-base-images-for-docker-users#)

**Review by gdams — APPROVED** [review]

**Comment by sophia-guo** [comment]
Thanks @gdams

**Pull request merged**